CMakeLists.txt

```diff
@@ -88,7 +88,8 @@ option(ENABLE_RDRAND                  "Enable RDRAND Hardware RNG Hash Seed."
 option(ENABLE_THREADING               "Enable partial threading support."                     OFF)
 option(OVERRIDE_GET_RANDOM_SEED       "Override json_c_get_random_seed() with custom code."   OFF)
 option(DISABLE_EXTRA_LIBS             "Avoid linking against extra libraries, such as libbsd." OFF)
-option(DISABLE_JSON_POINTER           "Disable JSON pointer (RFC6901) support."               OFF)
+option(DISABLE_JSON_POINTER           "Disable JSON pointer (RFC6901) and JSON patch support." OFF)
+option(DISABLE_JSON_PATCH             "Disable JSON patch (RFC6902) support."                 OFF)
```

Comment:
nice: thanks for these :)
i've been also meaning to do this;
i was a bit undecided whether to do them in CMake or classic Make

Reply:
er... actually _you_ did that: 393c98fb  :) 
I was rather surprised github allowed me to push to a branch in your fork.  Did you explicitly allow that?  
Anyway, it allowed for both of our changes to be rolled into this same PR, at the expense of a bit of confusion over the origin of the code, at least if you're just looking at the overall diffs.

Reply:
> Did you explicitly allow that?

no, i went with defaults;